src/views/boost/l10n.json

```diff
@@ -0,0 +1,34 @@
+{
+    "boost.headerText": "The {boostLink} kit brings your LEGO creations to life with powerful motors, a color sensor and more. By combining it with Scratch, you can build your own robotic creatures, tell physical-digital stories, invent new game controllers, or whatever else you can imagine.",
+    "boost.gettingStarted": "Getting Started",
+    "boost.connectingBoost": "Connecting BOOST to Scratch",
+    "boost.powerBoost": "Turn on your sensor by pressing the power button.",
+    "boost.useScratch3": "Use the {scratch3Link} editor.",
+    "boost.addExtension": "Add the BOOST extension.",
+    "boost.thingsToTry": "Things to Try",
+    "boost.makeAMotorMove": "Make a motor move",
+    "boost.findTurnMotorOnForSeconds": "Find the {turnMotorOnForSeconds} block and click on it.",
+    "boost.turnMotorOnForSeconds": "“turn motor A on for 1 seconds”",
```

Comment:
I notice you're using curly quote characters here. That's probably fine, but I wanted to flag it in case there's some reason we don't want to do that. I do see curly quotes used in other l10n strings elsewhere, so I imagine it's fine.

Reply:
Nope, don't use curly quotes. #1238
use normal ones escaped.

Reply:
Good catch- I'll address this as a followup (and talk to @chrisgarrity about it). thanks!